commands/dev-deploy.js

```diff
@@ -67,7 +67,7 @@ async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
         }
     }
 
-    const accountId = `dev-${Date.now()}`;
+    const accountId = `dev-${Date.now()}.test.near`;
```

Comment:
I think we should take `masterAccount` from config, suffix here not necessarily `test.near`.